model/project_ref_test.go

```diff
@@ -1138,54 +1138,54 @@ func TestFindOneProjectRefByRepoAndBranchWithPRTesting(t *testing.T) {
 	assert.NoError(doc2.Insert())
 
 	// repo doesn't have PR testing enabled, so no project returned
-	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "mine")
+	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "mine", patch.AutomatedCaller)
 	assert.NoError(err)
 	assert.Nil(projectRef)
 
 	repoDoc.Enabled = utility.TruePtr()
 	assert.NoError(repoDoc.Upsert())
-	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "mine")
+	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "mine", patch.AutomatedCaller)
 	assert.NoError(err)
 	assert.Nil(projectRef)
 
 	repoDoc.PRTestingEnabled = utility.TruePtr()
 	assert.NoError(repoDoc.Upsert())
-	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "mine")
+	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "mine", patch.AutomatedCaller)
 	assert.NoError(err)
 	require.NotNil(projectRef)
 	assert.Equal("defaulting_project", projectRef.Id)
 
 	// project PR testing explicitly disabled
 	doc.PRTestingEnabled = utility.FalsePtr()
 	assert.NoError(doc.Upsert())
-	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "mine")
+	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "mine", patch.AutomatedCaller)
 	assert.NoError(err)
 	assert.Nil(projectRef)
 
 	// project explicitly disabled
 	doc.Enabled = utility.FalsePtr()
 	doc.PRTestingEnabled = utility.TruePtr()
 	assert.NoError(doc.Upsert())
-	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "mine")
+	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "mine", patch.AutomatedCaller)
 	assert.NoError(err)
 	assert.Nil(projectRef)
 
 	// branch with no project doesn't work if repo not configured right
-	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "yours")
+	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "yours", patch.AutomatedCaller)
 	assert.NoError(err)
 	assert.Nil(projectRef)
 
 	repoDoc.RemotePath = "my_path"
 	assert.NoError(repoDoc.Upsert())
-	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "yours")
+	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "yours", patch.AutomatedCaller)
 	assert.NoError(err)
 	assert.NotNil(projectRef)
 	assert.Equal("yours", projectRef.Branch)
 	assert.True(projectRef.IsHidden())
 	firstAttemptId := projectRef.Id
 
 	// verify we return the same hidden project
-	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "yours")
+	projectRef, err = FindOneProjectRefByRepoAndBranchWithPRTesting("mongodb", "mci", "yours", patch.AutomatedCaller)
```

Comment:
Can you write a new test case for ManualCaller?

Reply:
done